DeviceSDK/src/main/java/org/astarteplatform/devicesdk/util/UUID5.java

```diff
@@ -0,0 +1,54 @@
+package org.astarteplatform.devicesdk.util;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Objects;
+import java.util.UUID;
+
+public class UUID5 {
```

Comment:
where this code is coming from? any license or author?
We must remember to document every time some code is copied from a 3rd party project, in order to let us track it and check/keep compliance with any license.
I would expect this information in both the commit message and the PR description.